.changeset/slimy-parks-begin.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Was this necessary because package.jsons changed? My pure repo .vscode change yesterday didn't seem to complain about no changeset, though I did preemptively add the `no changelog` label out of habit - anyway a label to pass the check might be nice :D